monai/utils/enums.py

```diff
@@ -233,3 +238,11 @@ class CommonKeys:
     LABEL = "label"
     PRED = "pred"
     LOSS = "loss"
+
+
+class DataObjects:
+    """Common classes used for arrays/tensors and then their usage in dict/mappings."""
+
+    Images = Union[torch.Tensor, np.ndarray]
```

Comment:
I think we already have such data type?
https://github.com/Project-MONAI/MONAI/blob/dev/monai/config/type_definitions.py#L70

Thanks.